### ISSUE TITLE
Add CommonLibrary.NET

### ISSUE THREAD

**Issue by JoshClose** [joelverhagen/NCsvPerf#19]
I don't think this has been updated in a long time, but people are recommending it on StackOverflow.

https://archive.codeplex.com/?p=commonlibrarynet

**Comment by JoshClose** [comment]
@joelverhagen Apparently this wasn't built in release mode. There are 2 versions on NuGet and both are the same.

![image](https://user-images.githubusercontent.com/114971/106178304-38f06c80-615f-11eb-98a5-ec8f5a442932.png)

Do you want to disable optimization validation, create a new build based on source, or forget this library altogether? Looks like the only source is on CodePlex Archive.

**Comment by JoshClose** [comment]
Some people have copied sources to github that could be used.

**Comment by joelverhagen** [comment]
Hm... my goal for this project is to be a helpful resource to the .NET community. In light of that, I think the most "net positive" outcome is to used an existing fork of this library on GitHub and test that one if it is clearly more popular than the debug version you've found. If there's no clear winner or if it looks fishy/very deviated from the upstream, then maybe a new fork is okay? I have already done this with FastCsvParser to get around a DLL name collision:
https://github.com/joelverhagen/Knapcode.FastCsvParser

In short, I think we should test the copy/version that users are most likely to discover and use.

But I'm not sure! What do you think?

**Comment by JoshClose** [comment]
I've tested like 4 or 5 of them and they're all the same. They're all an older version of .NET too, so it might be nice to be able to have an updated build for that also.

> Package 'CommonLibrary.NET 0.9.6.1' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.

I have all the unit tests passing for it, so I'll commit the code and you can create a fork if you like and change the package reference to that.

I sent an email to the maintainer of `CommonLibrary.NET` on NuGet and asked if he could release a new version in release mode. We'll see if he replies I guess.

**Comment by JoshClose** [comment]
https://github.com/joelverhagen/NCsvPerf/pull/24
